udata_hydra/crawl.py

```diff
@@ -44,6 +44,43 @@ async def insert_check(data: dict):
     return last_check["id"]
 
 
+async def send_check_data(check_data, last_check):
```

Comment:
I'm not 100% clear on the logic here, but you just moved it around, so let's keep that for later.